geomesa-ui/src/main/ui/src/app/map/map.js

```diff
@@ -1,39 +1,43 @@
 angular.module('geomesa.map', [])
 
-    .directive('geomesaMap', [function () {
+    .directive('geomesaMap', ['$http', function ($http) {
+
         return {
             restrict: 'E',
             scope: {
                 map: '=?',
                 api: '=',
+                cards: '=',
                 selectedPoint: '='
             },
-            link: function (scope, element, attrs) {
-                var baseLayer = L.tileLayer.provider('Stamen.TonerLite'),
-                    //wmsLayer = L.tileLayer.wms("http://dgeo:8080/geoserver/sf/wms", {
-                    wmsLayer = L.tileLayer.wms("localhost:8080/wcs/cite/wms", {
-                        layers: 'cite::QuickStart',
-                        format: 'image/png',
-                        transparent: true
-                    });
 
-                scope.map = L.map(element[0], {
-                    //center: L.latLng(40.279957,-74.73862),
-                    center: L.latLng(-38.09, -76.85),
-                    zoom: 8,
-                    maxZoom: 18,
-                    minZoom: 3,
-                    attributionControl: false,
-                    layers: [baseLayer, wmsLayer]
-                });
+            link: function (scope, element, attrs) {
+                scope.cards = [
+                    {
+                        Start: 'Click a data point to view it\'s attributes.'
+                    }
+                ];
 
-                scope.map.on('click', function (evt) {
-                    scope.$apply(function () {
-                        scope.selectedPoint = {
-                            lat: evt.latlng.lat,
-                            lng: evt.latlng.lng
-                        };
+                var baseLayer = new ol.layer.Tile({
+                        source: new ol.source.MapQuest({layer: 'osm'})
+                    }),
+                    wmsSource = new ol.source.TileWMS({
+                        url: 'http://geomesa:8080/geoserver/geomesa/wms',
+                        params: {LAYERS: 'QuickStart'}
+                    }),
+                        wmsLayer = new ol.layer.Tile({
+                        source: wmsSource
+                    }),
+                    olView = new ol.View({
+                        center: ol.proj.transform([37.41, 8.82], 'EPSG:4326', 'EPSG:3857'),
+                        zoom : 4,
+                        maxResolution : 40075016.68557849 / screen.width,
                     });
+
+                scope.map = new ol.Map({
```

Comment:
I had originally only used 'maxResolution' to get this effect, and just now added 'extent' to the tile layers because it was being buggy when the user would slide the map off the page such that they would get a new copy of the map rotated in from the other side. Just as a small note, we are using open layers 3 and there is no maxExtent in map anymore.